packages/DraftEditor/src/DraftPlugins/index.tsx

```diff
@@ -0,0 +1,114 @@
+import createAlignmentPlugin from "draft-js-alignment-plugin";
+import "draft-js-alignment-plugin/lib/plugin.css";
+import {
+  BlockquoteButton,
+  BoldButton,
+  CodeBlockButton,
+  CodeButton,
+  HeadlineOneButton,
+  HeadlineThreeButton,
+  HeadlineTwoButton,
+  ItalicButton,
+  OrderedListButton,
+  UnderlineButton,
+  UnorderedListButton
+} from "draft-js-buttons";
+import createBlockDndPlugin from "draft-js-drag-n-drop-plugin";
+import createFocusPlugin from "draft-js-focus-plugin";
+import "draft-js-focus-plugin/lib/plugin.css";
+import createImagePlugin from "draft-js-image-plugin";
+import "draft-js-image-plugin/lib/plugin.css";
+import createInlineToolbarPlugin from "draft-js-inline-toolbar-plugin";
+import "draft-js-inline-toolbar-plugin/lib/plugin.css";
+import createLinkifyPlugin from "draft-js-linkify-plugin";
+import "draft-js-linkify-plugin/lib/plugin.css";
+import { composeDecorators } from "draft-js-plugins-editor";
+import createResizeablePlugin from "draft-js-resizeable-plugin";
+import React, { useEffect } from "react";
+
+const focusPlugin = createFocusPlugin();
+const blockDndPlugin = createBlockDndPlugin();
+const linkifyPlugin = createLinkifyPlugin();
+const resizeablePlugin = createResizeablePlugin();
+const alignmentPlugin = createAlignmentPlugin();
+const inlineToolbarPlugin = createInlineToolbarPlugin();
+
+const { InlineToolbar } = inlineToolbarPlugin;
+const { AlignmentTool } = alignmentPlugin;
```

Comment:
Maybe you could do:
`const { InlineToolbar } = createInlineToolbarPlugin();
const { AlignmentTool } = createAlignmentPlugin();`
in lines 33 and 34,
and then remove lines 36 and 37?

Reply:
I'm reusing the variables